src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx

```diff
@@ -324,6 +330,15 @@ const TokensMoveDialogForm = ({
           )}
         </div>
       </DialogSection>
+      <DialogSection>
+        <div className={styles.textAreaSection}>
+          <Textarea
```

Comment:
Please add a character limit of `4000` to this textarea, sorry I missed it the first time around.

You can do this by passing the `maxLength` prop.

Eg:
```
maxLength={4000}
```